packages/core/src/behaviors.ts

```diff
@@ -107,9 +107,10 @@ export function spawnBehavior<TEvent extends EventObject, TEmitted>(
   const observers = new Set<Observer<TEmitted>>();
   const mailbox: TEvent[] = [];
   let flushing = false;
+  let done = false;
 
   const flush = () => {
-    if (flushing) {
+    if (flushing || done) {
```

Comment:
I made it so that a behavior cannot be flushed after its stopped. I think I might also have to make sure the mailbox cannot be filled after its done since that might be a memory leak.